streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/demo/HybridStreamTest.java

```diff
@@ -34,9 +40,12 @@ public boolean filter(Object value) throws Exception {
     }
   }
 
-  @Test
-  public void testHybridDataStream() throws InterruptedException {
+  @Test(timeOut = 60000)
+  public void testHybridDataStream() throws Exception {
     Ray.shutdown();
```

Comment:
use @beforemthod for ```Ray.shutdown()```

Reply:
`HybridStreamTest` has only one test, it's not necessary to add `@beforemthod`. If we have multiple tests, we do need to add `@beforemthod`